content/deploy/community-cloud/manage-your-app/edit-your-app.md

```diff
@@ -15,72 +15,69 @@ Spin up a cloud-based development environment for your deployed app in minutes.
 
-1. From your workspace at <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a>, click the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) next to your app. Click "**Edit**."
+1. From your workspace at <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a>, click the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) next to your app. Click "**Edit with Codespaces**."
 
    ![Edit your app with GitHub Codespaces](/images/streamlit-community-cloud/workspace-app-edit.png)
 
-2. A `.devcontainer/devcontainer.json` file will be added to your repository. If you already have a file of the same name in your repository, it will not be changed. You may delete or rename your existing devcontainer configuration if you would like your repository to receive the instance created by Streamlit Community Cloud.
+   Community Cloud will add a `.devcontainer/devcontainer.json` file to your repository. If you already have a file of the same name in your repository, it will not be changed. You may delete or rename your existing devcontainer configuration if you would like your repository to receive the instance created by Community Cloud.
 
-   ![Adding a devcontainer file to your repository](/images/streamlit-community-cloud/workspace-app-edit-preparing.png)
+1. Wait for GitHub to set up your codespace.
 
-3. Click "**Create codespace**" to confirm the creation of a codespace on your account. Read more about <a href="https://github.com/features/codespaces" target="_blank">GitHub Codespaces</a> to learn about monthly limits for free use and paid plans.
+   It can take several minutes to fully initialize your codespace. After you see the Visual Studio Code editor in your codespace, it can take several minutes to install Python and start the Streamlit server. When complete, you will see a split screen view with a code editor on the left and a running app on the right. The code editor opens two tabs by default: the repository's readme file and the app's entrypoint file.
 
-   ![Create your GitHub Codespace](/images/streamlit-community-cloud/deploy-codespaces-2.png)
+   ![Your new GitHub Codespace](/images/streamlit-community-cloud/deploy-template-blank-codespace.png)
 
-4. Wait for GitHub to set up your codespace.
+1. (Optional) Open the app preview in another tab for more room to work.
 
-   ![GitHub preparing your codespace](/images/streamlit-community-cloud/deploy-codespaces-3.png)
+   If you have multiple monitors and would like a little more room to work, you can open your app preview in another tab instead of using the Simple Browser with Visual Studio code. Just copy the URL from the Simple Browser into another tab, then close the Simple Browser. Now you have more room to edit your code. This page will continue to refer to your code on the left and your preview on the right.
```

Comment:
```suggestion
   If you have multiple monitors and want a little more room to work, open your app preview in another tab instead of using the Simple Browser with Visual Studio code. Just copy the URL from the Simple Browser into another tab, and then close the Simple Browser. Now you have more room to edit your code. This page will continue to display your code on the left and your preview on the right.
```